Gemfile

```diff
@@ -22,6 +22,7 @@ gem "omniauth-microsoft-office365"
 gem 'devise_saml_authenticatable'
 gem 'omniauth-rails_csrf_protection'
 gem "view_component"
+gem 'rails-controller-testing'
```

Comment:
This gem brings back deprecated methods removed in rails 5 in controller testing.

User rspec https://github.com/rspec/rspec-rails methods
```suggestion
```